src/App.tsx

```diff
@@ -14,11 +15,171 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Props = {};
+
+type State = {
+  goods: string[],
```

Comment:
You don't need to store goods array in `state`. You already have `goodFromServer` for this